change/@microsoft-teams-js-ad46a5c7-5cc0-4176-8539-0b126be3e1f5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix bug that when calling both video.registerForVideoFrame and video.mediaStream.registerForVideoStream an error happens: \"stream id is already registered\"",
```

Comment:
>Fix bug that when calling both video.registerForVideoFrame and video.mediaStream.registerForVideoStream an error happens: \"stream id is already registered\" [](http://example.com/codeflow?start=14&length=157)

Since this goes into the public changelog, we have a particular format (past tense and fixed with symbol names).

I recommend something like this:

Updated message sent to host by `video.registerForVideoFrame` to avoid duplication with message sent by `video.mediaStream.registerForVideoStream` (bug would manifest in \"stream id is already registered\" error message)

Reply:
updated as suggested